hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java

```diff
@@ -36,9 +53,16 @@
  */
 @Singleton
 public class FileGetContentsHandler extends PaidQueryHandler {
+    private final NetworkInfo networkInfo;
+
     @Inject
     public FileGetContentsHandler() {
-        // Exists for injection
+        // TODO: Not sure how to get the network info here.
+        this.networkInfo = null;
+    }
+
+    public FileGetContentsHandler(@NonNull final NetworkInfo networkInfo) {
```

Comment:
Can we just add @Inject at this constructor and remove the other one? I assume that It would complile.

Reply:
done thanks